setup.py

```diff
@@ -47,5 +42,11 @@
     keywords='terminal spin spinner',
     packages=['pyspin'],
     include_package_data=True,
-    install_requires=requires
```

Comment:
This was an attempt to require the newer version of setuptools, which was not successful